### PR TITLE
created a recipe for jamotools

### PR DESCRIPTION
**Pull request by Daniel-Cunningham**
Checklist

- [X] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [X] Source is from official source
- [X] Package does not vend other packages
- [X] Build number is 0
- [X] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your
      recipe (see
      [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos)
      for more details)
- [X] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by Daniel-Cunningham** [comment]
@adbeda Would you like to be a maintainer for Jamotools?

**Comment by adbeda** [comment]
yes!

File: recipes/jamotools/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    - jamotools.vector
+
+about:
+  home: "https://github.com/HaebinShin/jamotools"
+  license: "GPL"
+  license_family: "GPL"
+  license_file: "LICENSE"
+  summary: "A library for Korean Jamo split and vectorize."
+  doc_url: ""
+  dev_url: ""
```
Comment (marcelotrevisani) [inline recipes/jamotools/meta.yaml]: Could you please remove those empty attributes (``doc_url`` and ``dev_url``) ?

File: recipes/jamotools/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+test:
+  imports:
+    - jamotools
+    - jamotools.jamo
+    - jamotools.vector
+
+about:
+  home: "https://github.com/HaebinShin/jamotools"
+  license: "GPL"
```
Comment (marcelotrevisani) [inline recipes/jamotools/meta.yaml]: I think it should be:
```suggestion
  license: "GPL-3.0"
```

File: recipes/jamotools/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv "
+
+requirements:
+  host:
+    - future
+    - numpy
```
Comment (scopatz) [inline recipes/jamotools/meta.yaml]: Is numpy really a host dep?  It doesn't look like you are compiling against it.

**Comment by scopatz** [comment]
Thanks @Daniel-Cunningham

**Pull request merged**